crypto/evp/keymgmt_lib.c

```diff
@@ -18,12 +18,14 @@ static OSSL_PARAM *paramdefs_to_params(const OSSL_PARAM *paramdefs)
 {
     size_t cnt;
     const OSSL_PARAM *p;
-    OSSL_PARAM *params, *q;
+    OSSL_PARAM *params= NULL, *q;
```

Comment:
please put a space on both sides of =

Reply:
Hi @p-steuer ,

Just now modified requested changes, Please review now.

Thanks
Jayaram Matta